runtime/etc/mli/mli_client_runtime.c

```diff
@@ -52,6 +55,14 @@ void chpl_mli_client_deinit(struct chpl_mli_context* client) {
   return;
 }
 
+static void spawn_server() {
+  char command[256] = "./";
```

Comment:
Do we know this will never overflow? Just making sure.

Reply:
You're right, I should probably be doing a check to make sure it is safe